debian/12/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:12
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN set -x \
+	&& apt-get update && apt-get install -y --no-install-recommends \
+		ruby \
+		ruby-dev \
+		gcc \
+		make \
+		ca-certificates \
+		libffi-dev \
+		ruby-ffi \
```

Comment:
Lets clean up after the apt-get (similar to the `yum clean all` above)

https://docs.docker.com/develop/develop-images/instructions/#apt-get